vowpalwabbit/fb_parser/src/parse_example_flatbuffer.cc

```diff
@@ -271,8 +368,36 @@ bool get_namespace_hash(VW::workspace* all, const Namespace* ns, uint64_t& hash)
   return false;
 }
 
+bool features_have_names(const Namespace& ns)
+{
+  return flatbuffers::IsFieldPresent(&ns, Namespace::VT_FEATURE_NAMES) && (ns.feature_names()->size() != 0);
+  // TODO: It is not clear what the right answer is when feature_values->size is 0
```

Comment:
if the name collection is present at all and greater than 0 in length, it should match the value collection lenght for things to be rational